# 
# @brief    : custom message to start the folloewer uavs, name="takeOffCommand.msg"
# @time     : 2020/08/28
# @auhtor   : liuxu <liuxu.ccc@gmail.com> 
# 

#Information about component grabbing

std_msgs/Header header

uint64 timestamp   #timestamp
uint8  value #whether grab the box